ngx_http_fancyindex_module.c

```diff
@@ -1390,7 +1406,11 @@ ngx_http_fancyindex_cmp_entries_name_asc(const void *one, const void *two)
     ngx_http_fancyindex_entry_t *first = (ngx_http_fancyindex_entry_t *) one;
     ngx_http_fancyindex_entry_t *second = (ngx_http_fancyindex_entry_t *) two;
 
-    return (int) ngx_strcmp(first->name.data, second->name.data);
+    if(case_sensitive) {
+        return (int) ngx_strcmp(first->name.data, second->name.data);
+    } else {
+        return (int) ngx_strcasecmp(first->name.data, second->name.data);
+    }
```

Comment:
Same here: either:

```suggestion
    if (case_sensitive) {
        return (int) ngx_strcmp(first->name.data, second->name.data);
    } else {
        return (int) ngx_strcasecmp(first->name.data, second->name.data);
    }
```

or:

```suggestion
    if (case_sensitive)
        return (int) ngx_strcmp(first->name.data, second->name.data);
    else
        return (int) ngx_strcasecmp(first->name.data, second->name.data);
```